### PR TITLE
Fix space in code, add svg partials

### PR DESCRIPTION
**Pull request by Kseniia-FS**

### CODE REVIEW

File: backend/src/js/addToCartFromThumbs.js [hunk 1]
```diff
@@ -54,38 +47,28 @@ function addProductToCart(e) {
...
 
     }
-
-
 }
 
 // Listener for loading more products
-async function loadMore(e) {
+async function loadMore() {
     const page = Number(window.location.search.slice(6)[0]);
```
Comment (maksymsaltykov) [inline backend/src/js/addToCartFromThumbs.js]: Wrong approach that will lead to lots of bugs in the future. Change the way retrieving the page

File: backend/src/js/compare.js [hunk 2]
```diff
@@ -38,6 +38,16 @@ if (!storage || storage === []) {
 
 
         if (e.target.tagName.toLowerCase() === 'button') {
+
```
Comment (maksymsaltykov) [inline backend/src/js/compare.js]: Remove this empty line

File: backend/src/js/favorite.js [hunk 3]
```diff
@@ -37,6 +37,17 @@ if (!storage || storage === []) {
         const productID = e.target.parentElement.getAttribute("data-name");
 
         await fetch(`http://localhost:5000/productID/${productID}`).then(res => res.json()).then(data => {
+
```
Comment (maksymsaltykov) [inline backend/src/js/favorite.js]: remove 2 empty lines

File: backend/src/js/favorite.js [hunk 4]
```diff
@@ -37,6 +37,17 @@ if (!storage || storage === []) {
...
+
+            const storage = cart.getItem("cart");
+
+            if (storage) {
+                const parsedStorage = JSON.parse(storage);
+
+                orders = [...parsedStorage];
+
```
Comment (maksymsaltykov) [inline backend/src/js/favorite.js]: remove this empty line

File: backend/src/js/favorite.js [hunk 5]
```diff
@@ -37,6 +37,17 @@ if (!storage || storage === []) {
...
+            }
+
             const { image, title, price } = data.data;
```
Comment (maksymsaltykov) [inline backend/src/js/favorite.js]: remove this empty line

File: backend/src/js/filter.js [hunk 6]
```diff
@@ -58,6 +58,7 @@ function filterByPrice(e) {
 
 
     if (allProductsBtn.includes("All")) {
+
```
Comment (maksymsaltykov) [inline backend/src/js/filter.js]: remove this empty line and have a better recheck of this point through all your project

File: backend/src/js/shoppingCart.js [hunk 7]
```diff
@@ -11,56 +10,42 @@ const cartTable = document.querySelector("#cart-table");
...
 const chekoutBtn = document.querySelector("#checkoutBtn");
 
-
-
-
 // Rendering products
-
 if (!storage) {
-    cartTable.innerHTML = `<h2 style="text-align: center; margin-bottom: 200px; font-size: 42px; color: var(--accent-color); font-weight: bold;">Ooops, you didn't add any product!</h2>`;
+    cartTable.innerHTML = `<h2 style="text-align: center; margin-bottom: 200px;
```
Comment (maksymsaltykov) [inline backend/src/js/shoppingCart.js]: move it to the template

Comment (maksymsaltykov) [reply backend/src/js/shoppingCart.js]: avoid inline styles

**Review by maksymsaltykov — CHANGES_REQUESTED** [review]

File: backend/src/js/shoppingCart.js (hunk 7 above)
Comment (Kseniia-FS) [reply backend/src/js/shoppingCart.js]: changes added

**Review by maksymsaltykov — APPROVED** [review]

**Pull request merged**